libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/useDrillTargetTypeItems.ts

```diff
@@ -51,13 +58,19 @@ export const useDrillTargetTypeItems = (disableDrillDown?: boolean): IDrillTarge
             disableTooltipMessage: disableDrillDown
                 ? intl.formatMessage(messages.disableDrillDownToolTip)
                 : undefined,
+            tooltipMessage: messages.drilldownTooltip.id,
+            documentUrl:
+                "https://www.gooddata.com/docs/cloud/create-dashboards/drilling-in-dashboards/set-drill-down/",
```

Comment:
Seems like this document is not existing, please confirm with Petr Kaspa that we will not show the link until we have the link!

Thank you so much!

Reply:
It will be hidden until the link is available. Thanks!